backend/dataall/core/environment/cdk/pivot_role_stack.py

```diff
@@ -36,7 +41,14 @@ def generate_policies(self) -> List[iam.ManagedPolicy]:
             statements.extend(service.get_statements(self))
             logger.info(f'statements: {str(service.get_statements(self))}')
 
-        statements_chunks = split_policy_statements_in_chunks(statements)
+        statements_json = [statement.to_json() for statement in statements]
+        statement_chunks_json = split_policy_statements_in_chunks(statements_json)
+        statements_chunks = []
```

Comment:
Can we move the logic between 46 and 51 into the iam_cdk_utils? As a user I want a utility to convert a already-chunked json policy to cdk, I don't need to know about conditions or resources...

Reply:
Oh yes. Will do